packages/theme/components/checkbox.ts

```diff
@@ -0,0 +1,32 @@
+import { checkboxAnatomy } from '@chakra-ui/anatomy'
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+import { mode } from '@chakra-ui/theme-tools'
+
+const { definePartsStyle, defineMultiStyleConfig } =
+    createMultiStyleConfigHelpers(checkboxAnatomy.keys)
+
+const baseStyle = definePartsStyle({
+    // define the part you're going to style
+    control: {
+        border: "1px solid",
+        borderRadius: "2px",
+        borderColor: mode('gray.300', 'gray.25'),
```

Comment:
I have a same problem as Radio, border seems missing but it actually isn't and it works if I define "color" but it also changes border weirdly?

So I'm not sure what's going on here.

Reply:
Maybe what we thought `border` isn't actually a border...
But let's move on.